xili-tidy-tags/xili-tidy-tags.php

```diff
@@ -498,14 +485,24 @@ function xili_get_object_terms( $object_ids, $taxonomies, $args = array() ) {
 	$args = array_merge ( $defaults, $args );
 	extract ($args);
```

Comment:
By running extract after the validation of $taxonomies and $object_ids, they can be overwritten. I propose to validate them after extract.